scimath/StatsFramework/StatisticsAlgorithmQuantileComputer.h

```diff
@@ -69,7 +69,7 @@ class StatisticsAlgorithmQuantileComputer {
     // FIXME make protected once refactor is complete
     void _setSortedArray(const std::vector<AccumType>& v) { _sortedArray = v; }
 
-    void setMedian(CountedPtr<AccumType> median) { _median = median; }
+    void setMedian(std::shared_ptr<AccumType> median) { _median = median; }
```

Comment:
You can avoid copying the shared pointer here:
```suggestion
    void setMedian(std::shared_ptr<AccumType> median) { _median = std::move(median); }
```

Reply:
done